File: src/main/java/com/now/nowbot/throwable/LogException.java
package com.now.nowbot.throwable;

public class LogException extends RuntimeException{
    Throwable x;
    public LogException(String msg,Throwable throwable){
        super(msg);
        x = throwable;
    }

    public LogException(String msg) {
        super(msg);
    }
    public Throwable getThrowable(){
        return x;
    }
}

File: src/main/java/com/now/nowbot/throwable/botRuntimeException/LogException.kt
package com.now.nowbot.throwable.botRuntimeException

class LogException : RuntimeException {
    var throwable: Throwable? = null

    constructor(msg: String?, throwable: Throwable?) : super(msg) {
        this.throwable = throwable
    }

    constructor(msg: String?) : super(msg)
}
